feelpp/contrib/CMakeLists.txt

```diff
@@ -693,7 +701,7 @@ if ( FEELPP_ENABLE_TASKFLOW )
   message(STATUS "[feelpp-taskflow] cmake dir: ${CMAKE_INSTALL_DATADIR}/feel/cmake/modules")
   FetchContent_MakeAvailable(taskflow)
   add_library(feelpp_taskflow INTERFACE)
-  set_target_properties(Taskflow PROPERTIES OUTPUT_NAME "feelpp_taskflow")
+  #set_target_properties(Taskflow PROPERTIES OUTPUT_NAME "feelpp_taskflow")
```

Comment:
why is it related to the eigen3 dependency issue ?

Reply:
I usually rename the libraries to avoid conflicts on the filesystem if there is also an installation of the library

Reply:
it's not related. I worked on debian11 and cmake give an error because target Taskflow is an interface lib (and so no OUTPUT_NAME)